### PR TITLE
Update README with links, new commands, and update instructions

### PR DESCRIPTION
**Pull request by rido-min**
Updates README with 

- instructions to update/uninstall dmr-client
- new added commands (expand, index)
- add links to AzSDK client libraries

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,52 +88,110 @@ curl -L https://aka.ms/install-dmr-client-linux | bash
 iwr https://aka.ms/install-dmr-client-windows -UseBasicParsing | iex
 ```
 
+#### Update / Uninstall
+
+To update the client you must uninstall the current version first:
+
+```
+dotnet tool uninstall -g microsoft.iot.modelsrepository.commandline
```
Comment (digimaun) [inline README.md:96]: Good idea to show how to uninstall. Should we include guidance to uninstall beta.1? IIRC the package Id was `dmr-client`

File: README.md [hunk 2]
```diff
@@ -88,52 +88,110 @@ curl -L https://aka.ms/install-dmr-client-linux | bash
...
 
 #### Strict validation
 
 The Device Model Repo includes additional [requirements](pr-reqs.md), these can be validated with the `strict` flag.
 
 ```bash
-dmr-client validate --model-file ./my/model/file.json --repo . --strict true
+# Validate a model file using the DTDL parser checking dependencies with the current folder as a local repo in strict mode
+
+> dmr-client validate --model-file ./my/model/file.json --repo . --strict true
```
Comment (digimaun) [inline README.md:138]: you don't need to explicitly pass `--strict true` , `--strict` should be equivalent.

File: README.md [hunk 3]
```diff
@@ -88,52 +88,110 @@ curl -L https://aka.ms/install-dmr-client-linux | bash
...
+
+```bash
+# The default --local-repo value is the current directory. Be sure to specifiy the root for --local-repo.
+
+> dmr-client expand
 ```
 
 ## Consuming
 
+The *Models Repository Client* libraries are available in the Azure SDK for the next platforms:
```
Comment (digimaun) [inline README.md:184]: This sentence needs to be tweaked a bit.  Maybe something like

"There are Azure SDKs available for the models repository in the following languages: "

File: README.md (hunk 1 above)
Comment (rido-min) [reply README.md:96]: added a note

**Review by digimaun — APPROVED** [review]

**Pull request merged**